TriblerGUI/widgets/trustpage/js/radial_view/radial_link.js

```diff
@@ -90,6 +90,12 @@ function RadialLinks(svg, options) {
      * @param updateSelection
      */
     self.update = function (updateSelection) {
+        // Pass updated data through to all child elements using it
+        updateSelection.each(function (newData) {
```

Comment:
To make it future proof we make sure that all children of the link element also have the updated data.